scripts/setup.py

```diff
@@ -25,3 +26,23 @@
 
 with open(file, "w") as f:
     f.writelines(lines)
+
+file = os.path.join(
+    os.path.dirname(metatensor.torch.atomistic.__file__),
+    "units.py"
+)
+
+# At the moment, unit handling is incorrect in metatensor.torch.atomistic
+# This will fix it:
+with open(file, "r") as f:
+    lines = f.readlines()
+    for i, line in enumerate(lines):
+        if "return self._conversions[to_unit] / self._conversions[from_unit]" in line:
+            lines[i] = line.replace(
+                "return self._conversions[to_unit] / self._conversions[from_unit]",
+                "return self._conversions[from_unit] / self._conversions[to_unit]"
+            )
+            break
+
+with open(file, "w") as f:
+    f.writelines(lines)
```

Comment:
Are we hacking metatensor while installing metatensor models?

Reply:
No, the user has to run this, but it's documented (see export PR)